tests/test_vpc.rb

```diff
@@ -21,12 +21,13 @@
 # TestVpc - Minitest for Vpc node utility class
 class TestVpc < CiscoTestCase
   def setup
+    skip("Test not supported on #{node.product_id}") if platform != :nexus
```

Comment:
Better approach (now that #242 is merged) is to use the `@skip_unless_supported` property:

```diff
 class TestVpc < CiscoTestCase
+  @skip_unless_supported = 'vpc'
   def setup
```